src/main/webapp/profileScript.js

```diff
@@ -87,12 +87,19 @@ function loadRoutes() {
   fetch("/user-routes")
     .then((response) => response.json())
     .then((routesList) => {
+      console.log(routesList);
       for (i in routesList) {
         addRoute(routesList[i]);
       }
     });
 }
 
+var UserAccessType = {
+  OWNER: 1,
```

Comment:
As with the java comment, unless there is a very good reason enums should start at 0.